application-services/custom/cloud-event/res/configuration.yaml

```diff
@@ -0,0 +1,33 @@
+Writable:
+  LogLevel: "DEBUG"
+  StoreAndForward:
+    Enabled: false
+    RetryInterval: "5m"
+    MaxRetryCount: 10
+  Telemetry:
+    Interval: "0s" # Disabled since require MessageBus Trigger to be in use.
+
+Service:
+  Host: "localhost"
+  Port: 59780 # Adjust if running multiples examples at the same time to avoid duplicate port conflicts
+  StartupMsg: "Cloud Event Transforms"
+  RequestTimeout: "30s"
+  MaxResultCount: 0
```

Comment:
Remove as they are in common config.

Reply:
those values are different in common config, which is why I didnt remove them. For the other Edgex examples, should just remove them if the values are different?

Reply:
There is no reason they should be different